### PR TITLE
LPS-117939 Add app.bnd for Multi-Factor Authentication

### PR DESCRIPTION
**Pull request by martamedio**
Hey @csierra 

I have followed [this instructions](https://grow.liferay.com/group/guest/learn/-/wiki/34277/How+to+add+a+new+app+to+the+release+bundle) and I also created [the ticket to request the app.properties](https://issues.liferay.com/browse/LPS-117940).

Thanks!

### CODE REVIEW

**Comment by martamedio** [comment]
cc/ @gaborlovasdotcom

File: modules/dxp/apps/multi-factor-authentication/app.bnd [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Liferay-Releng-Bundle: true
+Liferay-Releng-Category: Security
+Liferay-Releng-Demo-Url:
+Liferay-Releng-Deprecated: false
+Liferay-Releng-Fix-Delivery-Method: core
+Liferay-Releng-Labs: false
+Liferay-Releng-Marketplace: true
+Liferay-Releng-Portal-Required: false
+Liferay-Releng-Public: ${liferay.releng.public}
+Liferay-Releng-Restart-Required: false
```
Comment (csierra) [inline modules/dxp/apps/multi-factor-authentication/app.bnd:12]: @gaborlovasdotcom do we have the poshi test to be able to set this to false? or should we mark it as true in the meantime?

Comment (csierra) [reply modules/dxp/apps/multi-factor-authentication/app.bnd:12]: it is listed here https://grow.liferay.com/people/App.bnd as a requirement

Comment (jpince) [reply modules/dxp/apps/multi-factor-authentication/app.bnd:12]: @csierra looks like no:
```
(master) $ git grep "hot.deploy.osgi.app.includes"
test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/CrossClusterReplicationES7.testcase:              property hot.deploy.osgi.app.includes = "portal-search-elasticsearch-cross-cluster-replication";
test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/Elasticsearch7EE.testcase:                property hot.deploy.osgi.app.includes = "portal-search-learning-to-rank";
test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:        
        property hot.deploy.osgi.app.includes = "saml";
```
And you can always change it to false later if Gabor doesn't have time to automate right now :).

**Comment by csierra** [comment]
Please see https://github.com/csierra/liferay-portal/pull/774

File: modules/dxp/apps/multi-factor-authentication/app.bnd (hunk 1 above)
Comment (gaborlovasdotcom) [reply modules/dxp/apps/multi-factor-authentication/app.bnd:12]: I have some question about this what I shared with you @csierra .

Comment (gaborlovasdotcom) [reply modules/dxp/apps/multi-factor-authentication/app.bnd:12]: As discussed with @xbrianlee, the test is not needed.

**Pull request closed without merging**